### PR TITLE
Add .travis.yml to run mypy.

### PR DESCRIPTION
**Pull request by progval**
Before merging this PR, you should:

1. Merge #191 
2. Login to https://travis-ci.org/ and enable the bitbot repo (at https://travis-ci.org/jesopo/bitbot )

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+cache: pip
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
+  - "3.8-dev"
+  - "nightly"
+install:
+  - pip install mypy -r requirements.txt
```
Comment (jesopo) [inline .travis.yml]: shouldn't this be `pip3`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+script:
+  - mypy --ignore-missing-imports src
```
Comment (jesopo) [inline .travis.yml]: `mypy --ignore-missing-imports ./bitbotd modules/`

File: .travis.yml (hunk 1 above)
Comment (progval) [reply .travis.yml]: it doesn't really matter on Travis. Changed anyway.

**Comment by jesopo** [comment]
merci

**Pull request merged**